test/Microsoft.ML.Tests/TrainerEstimators/MetalinearEstimators.cs

```diff
@@ -25,7 +25,7 @@ public void OVAWithAllConstructorArgs()
                 new AveragedPerceptronTrainer.Options { Shuffle = true });
 
             var ova = ML.MulticlassClassification.Trainers.OneVersusAll(averagePerceptron, imputeMissingLabelsAsNegative: true,
-                calibrator: calibrator, maxCalibrationExamples: 10000, useProbabilities: true);
+                calibrator: calibrator, maximumCalibrationExampleCount: 10000, useProbabilities: true);
```

Comment:
>maximumCalibrationExampleCount [](start = 40, length = 30)

Do we really need to expand out the names? I'm on the fence. Good for clarity, bad for length.

Reply:
Yes, we have many similar names in tree trainers.